tests/system_wide_config.rs

```diff
@@ -0,0 +1,47 @@
+use assert_cmd::cargo::CommandCargoExt;
+use predicates::{prelude::predicate, str::PredicateStrExt};
+use std::process::Command;
+
+fn bat_raw_command() -> Command {
+    let mut cmd = Command::cargo_bin("bat").unwrap();
+    cmd.current_dir("tests/examples");
+    cmd.env_remove("BAT_CACHE_PATH");
+    cmd.env_remove("BAT_CONFIG_DIR");
+    cmd.env_remove("BAT_CONFIG_PATH");
+    cmd.env_remove("BAT_OPTS");
+    cmd.env_remove("BAT_PAGER");
+    cmd.env_remove("BAT_STYLE");
+    cmd.env_remove("BAT_TABS");
+    cmd.env_remove("BAT_THEME");
+    cmd.env_remove("COLORTERM");
+    cmd.env_remove("NO_COLOR");
+    cmd.env_remove("PAGER");
+    cmd
+}
+
+fn bat() -> assert_cmd::Command {
+    assert_cmd::Command::from_std(bat_raw_command())
+}
```

Comment:
it would be great to move this duplicated(?) code to a common test-helper module